language/stdlib/modules/TravelRuleLimit.move

```diff
@@ -0,0 +1,39 @@
+address 0x0 {
+
+    module TravelRuleLimit {
```

Comment:
Also: can we call this `DualAttestationLimit`? This is the term that we use elsewhere, and the travel rule isn't the only use-case for payments with dual attestation.